smithy-model/src/main/java/software/amazon/smithy/model/transform/ModelTransformer.java

```diff
@@ -376,7 +376,26 @@ public Model removeUnreferencedTraitDefinitions(Model model, Predicate<Shape> ke
      * @return Returns the transformed model.base.
      */
     public Model scrubTraitDefinitions(Model model) {
-        return new ScrubTraitDefinitions().transform(this, model);
+        return scrubTraitDefinitions(model, FunctionalUtils.alwaysTrue());
+    }
+
+    /**
+     * Removes trait definitions from a model and all shapes that are
+     * only connected to the graph either directly or transitively by a
+     * trait definition shape.
+     *
+     * <p>This can be useful when serializing a Smithy model to a format that
+     * does not include trait definitions and the shapes used by trait definitions
+     * would have no meaning (e.g., Swagger).
```

Comment:
s/Swagger/OpenAPI

Reply:
Fixed this and a couple other Swagger references.